README.md

```diff
@@ -152,7 +152,7 @@ babili     21.9kB  219%    7.46kB 164%     2ms        722ms
 closure js 24.01kB 191%    8.04kB 144%     2ms        3301ms
 ```
 
-Run with: `./scripts/benchmark.js ./scripts/fixtures/backbone.js`
+Run with: `./scripts/benchmark.js backbone` **or** `./scripts/benchmark.js backbone backbone/backbone.js`
```

Comment:
Not sure exactly what you're getting at but I think you mean the latter part doesn't need to be there?  I just felt it would be helpful to make it obvious how you would run the script against a specific file.  However, the problem is is that I have not changed these results table so the results are actually for whatever files are in the fixtures folder.  But I didn't want to add confusion by having the first command which uses the main file and a second command which uses a different file than what the first does.  Now I think about it, though, maybe have two tables per package showing the main file results and a specific file results?